src/pages/index.js

```diff
@@ -84,7 +84,11 @@ const Home = (props) => {
                 </Typography>
             </div>
             <div
-                style={{ margin: "auto", maxWidth: 1168 }}
+                style={{
+                    margin: "auto",
+                    width: "100%",
+                    maxWidth: 790,
```

Comment:
Dropping the 70% and using an approximation of the observed width. This is the size of the middle column of the site that contains the cards.